tests/robustness/makefile.mk

```diff
@@ -124,9 +125,7 @@ $(GOPATH)/bin/gofail: tools/mod/go.mod tools/mod/go.sum
 	cd /tmp/etcd-release-3.5-failpoints/; \
 	  git clone --depth 1 --branch release-3.5 https://github.com/etcd-io/etcd.git .; \
 	  go get go.etcd.io/gofail@${GOFAIL_VERSION}; \
-	  (cd server; go get go.etcd.io/gofail@${GOFAIL_VERSION}); \
```

Comment:
Is there a reason that for `/tmp/etcd-v3.5.12-beforeSendWatchResponse/bin` we keep
```
(cd server; go get go.etcd.io/gofail@${GOFAIL_VERSION}); \
(cd etcdctl; go get go.etcd.io/gofail@${GOFAIL_VERSION}); \
(cd etcdutl; go get go.etcd.io/gofail@${GOFAIL_VERSION}); \ 
```
but we remove those for `/tmp/etcd-release-3.5-failpoints/bin`? :)

Reply:
We know that release branch has the `tools/mod` directory, however for tags we don't know. Some of them might have it, some don't.

Reply:
But we will always have `server`, `etcdctl`, and `etcdutl` directories, so we will always need to `go get` the `gofail` library for them, no? :)

Reply:
The build.sh will execute the `go get go.etcd.io/gofail@${GOFAIL_VERSION})` commands for all `servers, etcdutl, etcdctl and tests.`, so it's OK to remove the duplicated commands in makefile.

https://github.com/etcd-io/etcd/blob/d83c8bd1074bd238b3d806d0e25797ec87c6d7c8/build.sh#L33-L36